CHANGELOG.md

```
@@ -1,3 +1,36 @@
+# 3.5.2
+> Published 2 August 2026
+
+### Improvements
```

markdownlint-cli2 (0.23.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)